test/check-machines-create

```diff
@@ -1233,6 +1298,12 @@ vnc_password= "{vnc_passwd}"
                     b.set_input_text("#user-login", self.user_login)
                 if self.root_password:
                     b.set_input_text("#create-vm-dialog-root-password-pw1", self.root_password)
+                if self.ssh_keys is not None:
+                    for idx, key in enumerate(self.ssh_keys):
+                        b.click("button:contains(Add SSH keys)")
+                        b.set_input_text(f"#create-vm-dialog-ssh-key-{idx} textarea", key, value_check=False, blur=False)
+                        # Check that ssh key was validated
+                        b.wait_visible(f"#create-vm-dialog-ssh-key-{idx} #validated")
```

Comment:
Can you please check a broken validation as well? To cover the "ssh-keygen exits with non-zero" case?